src/service/video-manager-impl.js

```diff
@@ -623,6 +661,45 @@ class VideoEntry {
   }
 
   /**
+   * Gets the provided metadata and fills in missing fields
+   * @private
+   */
+  fillMetaData_() {
+    if (this.video.preimplementsMediaSessionAPI()) {
+      return;
+    }
+
+    if (this.video.getMetaData()) {
+      this.metaData_ = map(this.video.getMetaData());
+    }
+
+    if (!this.metaData_.artwork || this.metaData_.artwork.length == 0) {
+      const posterUrl = this.video.element.getAttribute('poster')
+                        || this.internalElement_.getAttribute('poster')
+                        || parseSchemaImage(this.ampdoc_)
+                        || parseOgImage(this.ampdoc_)
+                        || parseFavicon(this.ampdoc_);
+
+      if (posterUrl) {
+        this.metaData_.artwork = [{
+          'src': posterUrl,
+        }];
+      }
+    }
+
+    if (!this.metaData_.title) {
+      const title = this.video.element.getAttribute('title')
+                    || this.video.element.getAttribute('aria-label')
+                    || this.internalElement_.getAttribute('title')
```

Comment:
probably should not look into `internalElement`  (there shouldn't be any cases where inner element has has but not the custom element wrapper of it)